prisma-fmt/src/code_actions/field.rs

```diff
@@ -0,0 +1,84 @@
+use lsp_types::{CodeAction, CodeActionKind, CodeActionOrCommand};
+use psl::{
+    diagnostics::Span,
+    parser_database::walkers::{self, FieldWalker},
+    schema_ast::ast::{WithAttributes, WithName, WithSpan},
+};
+
+use super::CodeActionsContext;
+
+pub(super) fn add_missing_opposite_relation(
+    actions: &mut Vec<CodeActionOrCommand>,
+    context: &CodeActionsContext<'_>,
+    field: FieldWalker<'_>,
+) {
+    match field.refine() {
+        Some(walkers::RefinedFieldWalker::Relation(_)) => (),
+        _ => return,
+    }
+
+    let name = field.model().name();
+    let target_name = field.ast_field().field_type.name();
+    let diagnostics = context.diagnostics_for_span_with_message(
+        field.ast_field().span(),
+        "is missing an opposite relation field on the model",
+    );
+
+    if diagnostics.is_empty() {
+        return;
+    }
+
+    // * We know that this is safe to unwrap here as otherwise this diagnostic would be
+    // * replaced with the one that we check for in `fn create_missing_block_for_model`
+    let target_model = context.db.find_model(target_name).unwrap();
+
+    let target_file_id = target_model.file_id();
+    let target_file_content = context.db.source(target_file_id);
+
+    target_model.fields().last().unwrap();
```

Comment:
Why are we doing this?

Reply:
Which part specifically? @SevInf

Reply:
Only the line itself.  Seems like it's not doing anything except panicing if fields are empty?

Reply:
Ah, no I see what you mean now, I think I was using it at one point and then didn't need it. Forgot to remove 😅